File: BasicLevel_C++/1046. 划拳(15).cpp
#include <iostream>
using namespace std;
int main() {
    int n, jia = 0, yi = 0, jiahan, jiahua, yihan, yihua, sum;
    cin >> n;
    for (int i = 0; i < n; i++) {
        cin >> jiahan >> jiahua >> yihan >> yihua;
        sum = jiahan + yihan;
        if (sum == jiahua && sum != yihua)
            yi++;
        if (sum != jiahua && sum == yihua)
            jia++;
    }
    cout << jia << " " << yi;
    return 0;
}

File: BasicLevel_Golang/1046. 划拳(15).go
package main

import (
	"fmt"
)

func main() {
	var (
		N   int
		win [2]int
	)
	fmt.Scanln(&N)
	for i := 0; i < N; i++ {
		var jiahan, jiahua, yihan, yihua int
		fmt.Scanln(&jiahan, &jiahua, &yihan, &yihua)
		count := jiahan + yihan
		if jiahua == count && yihua != count {
			win[1]++
		}
		if yihua == count && jiahua != count {
			win[0]++
		}
	}
	fmt.Println(win[0], win[1])

}
